front/src/plugins/kakao-map.js

```diff
@@ -29,8 +29,18 @@ const KakaoMap = {
         level: 2,
       };
       this.map = new kakao.maps.Map(mapContainer, options);
+      this.clusterer = createClusterer.call(this);
     };
 
+    function createClusterer() {
```

Comment:
```suggestion
    const createClusterer = () => {
```
다른 메소드들처럼 arrow function으로 하면 어떨까요?